pkg/authorization/cache/policy_cache_test.go

```diff
@@ -0,0 +1,105 @@
+package cache
+
+import (
+	"testing"
+	"time"
+
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/labels"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+	testregistry "github.com/openshift/origin/pkg/authorization/registry/test"
+)
+
+func stop(channel chan struct{}) {
+	channel <- struct{}{}
+}
+
+func TestPolicyGet(t *testing.T) {
+	policyStop := make(chan struct{})
+	bindingStop := make(chan struct{})
+	defer stop(policyStop)
+	defer stop(bindingStop)
+
+	policyRegistry := testregistry.NewPolicyRegistry(testPolicies(), nil)
+	bindingRegistry := testregistry.NewPolicyBindingRegistry(testBindings(), nil)
+
+	policyCache := NewPolicyCache(bindingRegistry, policyRegistry)
+	policyCache.RunUntil(bindingStop, policyStop)
+
+	time.Sleep(50 * time.Millisecond)
```

Comment:
This will break on travis and other slow test envs - you should set something up to make a call a bunch of times until it returns what you expect (you can loop forever because -timeout)